frontend/src/common/backend.ts

```diff
@@ -195,6 +195,23 @@ export type VersionDto = {
   keycloakVersion: string;
 }
 
+export class LicenseStatusDto {
+  constructor(
+    public licensedSeats: number,
+    public usedSeats: number,
+    public expiresAt: Date) {
```

Comment:
`expiresAt` might be `null`:

```suggestion
    public expiresAt?: Date) {
```

Reply:
Fixed in f9c3f00f7f1d5197a054455571229cbe1ae3e716.